playbooks/common/openshift-nfs/config.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Set nfs facts
+  hosts: oo_nfs_to_config
+  roles:
+  - role: openshift_facts
+  post_tasks:
+  - openshift_facts:
+      role: nfs
+      local_facts:
+        exports_dir: "{{ openshift_nfs_exports_dir | default(None) }}"
+        export_options: "{{ openshift_nfs_export_options | default(None) }}"
+        registry_storage: "{{ openshift_nfs_registry_storage | default(None) }}"
+        registry_volume: "{{ openshift_nfs_registry_volume | default(None) }}"
+
+- name: Configure nfs hosts
+  hosts: oo_nfs_to_config
+  vars:
+    exports_dir: "{{ openshift.nfs.exports_dir }}"
+    export_options: "{{ openshift.nfs.export_options }}"
+    registry_storage: "{{ openshift.nfs.registry_storage }}"
+    registry_volume: "{{ openshift.nfs.registry_volume }}"
+  roles:
+  - role: openshift_storage_nfs
```

Comment:
We should be pulling those values out of the facts we set, instead of relying on the playbook level variables. That way we don't start creating artificial dependencies on different playbooks.